src/hooks/useFingerprint.ts

```diff
@@ -0,0 +1,16 @@
+'use client';
+import { useEffect, useState } from 'react';
+
+export const useFingerprint = () => {
+  const [fingerprint, setFingerprint] = useState<string>();
+  // Create a new ClientJS object
+
+  useEffect(() => {
+    // Get the client's fingerprint id;
+    const newFingerprint = 'abcd';
+    console.log(newFingerprint);
+    setFingerprint(`${newFingerprint}`);
+  }, []);
+
+  return fingerprint;
+};
```

Comment:
This should be used to include fingerprint. I tried client-js and get-browser-fingerprint, both doing the job, but will error messages in the log - so postponed this for now. Wdyt?